packages/core-mobile/e2e/pages/popUpModal.page.ts

```diff
@@ -184,13 +192,39 @@ class PopUpModalPage {
         by.id(popUpModalLoc.tokenGasFee)
       )
     }
-    console.log(`Current gas fee: ${tokenGasFee}`)
+
+    // Verify network fee is not over the estimated gas fee
+    console.log(`Current network fee: ${tokenGasFee}`)
     const tokenMatch = tokenGasFee?.match(/[\d.]+/)
     const token = tokenMatch ? parseFloat(tokenMatch[0]) : 0
     assert(
       token > 0 && token <= estimatedGasFee,
       `currVal: ${token} !<= expectedVal: ${estimatedGasFee}`
     )
+
+    // Verify base fee for C-Chain
+    if (isCChain) {
+      console.log(`Current base fee for C-Chain`)
+      const tolerance = 3
```

Comment:
you might want to do percentage here or else it might fail a lot

Reply:
good call! I've pushed a new commit